packages/cli/package.json

```diff
@@ -35,10 +35,10 @@
   "dependencies": {
     "@ast-grep/napi": "0.11.0",
     "@graphql-codegen/cli": "5.0.1",
-    "@oclif/core": "3.15.1",
-    "@shopify/cli-kit": "3.56.3",
+    "@oclif/core": "3.19.6",
+    "@shopify/cli-kit": "3.58.0",
     "@shopify/hydrogen-codegen": "^0.2.2",
-    "@shopify/oxygen-cli": "4.3.6",
+    "@shopify/oxygen-cli": "~4.4.0",
```

Comment:
@wizardlyhel Could i allow patch updates to the oxygen-cli? Or should i keep the version locked?